src/query/ts/exemplar.go

```diff
@@ -0,0 +1,10 @@
+package ts
+
+// Exemplar holds the exemplar information for a datapoint.
+type Exemplar []byte
+
+// Exemplars is an array of exemplars.
```

Comment:
nit: Maybe call this SeriesExemplar or something to give it some context that it's describing the set of exemplars for the entire series?